colossalai/context/moe_context.py

```diff
@@ -1,6 +1,9 @@
 import torch
 import torch.distributed as dist
-from .parallel_mode import ParallelMode
+
+from colossalai.context.parallel_mode import ParallelMode

```

Comment:
I think the old way is better, since it has less probability to cause reference error.
When you import colossalai.context.parallel_mode, it will call __init__.py in colossalai, 
this may cause reference problem.

Reply:
No... It won't.

Unfortunately, relative imports can be messy, particularly for shared projects where directory structure is likely to change. Relative imports are also not as readable as absolute ones, and it’s not easy to tell the location of the imported resources.